### PR TITLE
Add details in glossary (DG)

### LINKED ISSUE #269: Glossary in DG seems empty

### PR DESCRIPTION
**Pull request by OVReader**
Fix #269

### CODE REVIEW

**Review by ngdeqi — COMMENTED** [review]
Good identification of the terms that a reader may have misconceptions about.

**Review by FeliciaBeatrice — APPROVED** [review]
LGTM

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -542,6 +542,12 @@ ___
 ## Appendix D: Glossary
 
 * *client* - Person who is seeking for property to rent
+* *unit type* - Type of housing in Singapore (HDB, Condominium, Bungalow, etc)
+* *pairing* - If a client and property is paired, it implies that the client will be renting that property.
+* *unpair* - If a client and property is unpaired, it implies that the client is no longer renting any property.
+* *ui* - User Interface which is responsible for any interactions (read input / print messages) between application and user.
+* *storage* - Made up of plain text files storing client/property records.
```
Comment (wilsonngja) [inline docs/DeveloperGuide.md:549]: Thanks for adding the glossary :D 
It's really clear

**Review by wilsonngja — APPROVED** [review]
Good job! LGTM

**Pull request merged**